app/assets/stylesheets/skills_page.scss

```diff
@@ -0,0 +1,28 @@
+$base        : #aaaaaa;
+$secondary-color: #ffffff;
+
+.side-bar {
+  background-color: $secondary-color;
+  border: 25px $secondary-color solid;
+  border-radius: 6px;
+  margin-top: 20px;
+  padding: 0;
+}
+
+.fl-img-sidebar {
+  border-bottom-color: #000000;
```

Comment:
Color literals like `#000000` should only be used in variable declarations; they should be referred to via variable everywhere else.